doc/release/technotes/README.firstClassFns.rst

```diff
@@ -51,14 +59,18 @@ Lambda functions are anonymous first-class function objects. In other
 words, they are expressions rather than formally-defined named
 functions. They are available with the following syntax:
 
+.. code-block:: chapel
+
```

Comment:
Since this is pseudocode instead of chapel code, I wouldn't have it chapel highlighted